draft-ietf-quic-tls.md

```diff
@@ -1421,19 +1429,23 @@ amplification.
 
 ## Header Protection Analysis {#header-protect-analysis}
 
-Header protection relies on the packet protection AEAD being a pseudorandom
-function (PRF), which is not a property that AEAD algorithms
-guarantee. Therefore, no strong assurances about the general security of this
-mechanism can be shown in the general case. The AEAD algorithms described in
-this document are assumed to be PRFs.
-
-The header protection algorithms defined in this document take the form:
+{{NAN}} analyzes authenticated encryption algorithms which provide nonce
+privacy, referred to as "Hide Nonce" (HN) transforms. The general header
+protection construction in this document is one of those algorithms (HN1).
+Header protection uses the output of the packet protection AEAD to derive
+`sample`, and then encrypts the header field using a pseudorandom function
+(PRF) as follows:
 
 ~~~
 protected_field = field XOR PRF(hp_key, sample)
 ~~~
 
-This construction is secure against chosen plaintext attacks (IND-CPA) {{IMC}}.
+Assuming hp_key is distinct from the packet protection key, this construction
+(HN1) achieves AE2 security and therefore guarantees privacy of `field`, the
+protected packet header. One important distinction between HN1 and the header
+protection construction in this document is that the latter uses an AEAD
+algorithm as the PRF. However, since the encrypted output of an AEAD is
```

Comment:
I think the distinction is important because AEADs are (depending on one's definition) not PRFs. 

> We should probably say something about what it takes to define a new header protection scheme somewhere though.

Hmm, I'm not sure. That's probably best done elsewhere, e.g., a CFRG document, that we then cite.